src/inject/fallback.ts

```diff
@@ -0,0 +1,8 @@
+const fallbackStyle = document.createElement('style');
```

Comment:
This is more micro-code which I exactly want to elimate. It should be as optimized/fast as it can be. Also Linux(part of it) is suffering a bug that this can't be dark #2722

Reply:
I tried
```javascript
var start = performance.now();
var matches = matchMedia('(prefers-color-scheme: dark)').matches;
var linux = navigator.appVersion.includes('Linux');
var end = performance.now();
console.log(matches, linux, end - start);
```
and it gave `0.05ms` in Chrome and `0` in Firefox, so I think it's very fast. So use a check like (I have no Linux to check if it works)
```javascript
if (matchMedia('(prefers-color-scheme: dark)').matches || navigator.appVersion.includes('Linux')) {
    // ...
}
```

Reply:
There's also `navigator.platform`, I don't know if it is more reliable.

Reply:
Well, whatever chromium has some native dark page loading color when it's detected for `(prefers-color-scheme: dark)`

Also a low system `0.15ms` in chrome `2ms` in firefox. So hold in mind a normal 60fps(Chrome/Firefox renders aims to) will need to get this code injected within `16.66...ms` Know as the first frame evaluted the CSS and than painted.